PROJECT_GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+# Tinted Theming Project Governance
+
+Welcome to the Tinted Theming project! This document outlines the
+governance structure and procedures for maintaining and contributing to
+this theming driven community.
+
+## 1. Project Governance and Final Decision Authority
+
+The Tinted Theming project is currently governed by three admins:
```

Comment:
Do we want to mention former admins anywhere?

Reply:
I've added a new commit adding this list to the README.md. It seemed unrelated to project governance, what do you think?